File: CodigoDelitos/delitos/persona.java
package delitos;

//import delitos.direccion;

abstract public class persona {
	private String primerNombre;
	private String segundoNombre;
	private String primerApellido;
	private String segundoApellido;
	private direccion Direccion;
    private int edad;
    private String identidad;
	/////////////////////////////////////////////////////////////////
	private String PrimerNombre;
	private String SegundoNombre;
	private String PrimerApellido;
	private String SegundoApellido;
	
    private int Edad;
    ///////////////////////////////////////////////////////////
    //constructor
    public persona(){
	super();
    }
    //constructor  con parametros
	public persona(String primerNombre, String segundoNombre, String primerApellido, String segundoApellido,
			char genero, int edad, String identidad) {
		super();
		this.primerNombre = primerNombre;
		this.segundoNombre = segundoNombre;
		this.primerApellido = primerApellido;
		this.segundoApellido = segundoApellido;
		this.edad = edad;
		this.identidad= identidad;
		///////////////////////////////////////////////////////////////
		this.PrimerNombre = primerNombre;
		this.SegundoNombre = segundoNombre;
		this.PrimerApellido = primerApellido;
		this.SegundoApellido = segundoApellido;
		this.Edad = edad;
		///////////////////////////////////////////////////////////////////
	    
	}


	public String getPrimerNombre() {
		return PrimerNombre;
	}
	public void setPrimerNombre(String primerNombre) {
		PrimerNombre = primerNombre;
	}

	public String getSegundoNombre() {
		return SegundoNombre;
	}
	public void setSegundoNombre(String segundoNombre) {
		SegundoNombre = segundoNombre;
	}

	public String getPrimerApellido() {
		return PrimerApellido;
	}
	public void setPrimerApellido(String primerApellido) {
		PrimerApellido = primerApellido;
	}

	public String getSegundoApellido() {
		return SegundoApellido;
	}
	public void setSegundoApellido(String segundoApellido) {
		SegundoApellido = segundoApellido;
	
	}

	public int getEdad() {
		return Edad;
	}
	public void setEdad(int edad) {
		Edad = edad;
	}

    public direccion getDireccion() {
		 return Direccion;
		    }

    public void setDireccion(direccion direccion) {
		        this.Direccion = direccion;
		    }
	
	public String getIdentidad(){
		return identidad;
	}		
	public void setIdentidad(String identidad){
		this.identidad = identidad;
	}


public void mostrarInformacion() {
    System.out.println("PrimerNombre: " + primerNombre);
    System.out.println("SegundoNombre: " + segundoNombre);
    System.out.println("PrimerApellido: " + primerApellido);
    System.out.println("SegundoApellido: " + segundoApellido);
    System.out.println("Edad: " + edad);
   // getDireccion();
}

	
}

File: CodigoDelito/VersionWEB/php/Persona.php
<?php

include_once("Direccion.php");
class Persona extends Direccion
{


   private $PrimerNombre;
   private $PrimerApellido;
   private $SegundoNombre;
   private $SegundoApellido;
   private $Edad;
   private $Genero;
   //private direccion
   public $identidad;

   public $objDireccion;

   //constructor
   public function __construct($departamento, $municipio, $colonia, $numeroBloque, $codigoPostal, $numeroCasa, $PrimerNombre, $SegundoNombre, $PrimerApellido, $SegundoApellido, $Edad, $identidad, $Genero)
   {
      $this->PrimerNombre = $PrimerNombre;
      $this->SegundoNombre = $SegundoNombre;
      $this->PrimerApellido = $PrimerApellido;
      $this->SegundoApellido = $SegundoApellido;
      $this->Edad = $Edad;
      $this->identidad = $identidad;
      $this->Genero = $Genero;
      //objeto para direccion de las especializaciones de persona
      $this->objDireccion = new Direccion($departamento, $municipio, $colonia, $numeroBloque, $numeroCasa, $codigoPostal); 
      
   }


   //metodo para importar la direccion de la persona 
   /* public function obtenerDireccion(){
     $direction = $direccion->$
    }*/

public function getobjDireccion(){
  return $this->objDireccion;
}

   //getters 
   public function getPrimerNombre()
   {
      return $this->PrimerNombre;
   }
   public function getPrimerApellido()
   {
      return $this->PrimerApellido;
   }
   public function getSegundoNombre()
   {
      return $this->SegundoNombre;
   }
   public function getSegundoApellido()
   {
      return $this->SegundoApellido;
   }
   public function getEdad()
   {
      return $this->Edad;
   }
   public function getGenero()
   {
      return $this->Genero;
   }
   public function getIdentidad()
   {
      return $this->identidad;
   }


   //setters
   public function setPrimerNombre($PrimerNombre)
   {
      $this->PrimerNombre = $PrimerNombre;
      return $this;
   }
   public function setPrimerApellido($PrimerApellido)
   {
      $this->PrimerApellido = $PrimerApellido;
      return $this;
   }
   public function setSegundoNombre($SegundoNombre)
   {
      $this->SegundoNombre = $SegundoNombre;
      return $this;
   }
   public function setSegundoApellido($SegundoApellido)
   {
      $this->SegundoApellido = $SegundoApellido;
      return $this;
   }
   public function setEdad($edad)
   {
      $this->edad = $edad;
      return $this;
   }
   public function setGenero($Genero)
   {
      $this->Genero = $Genero;
      return $this;
   }
   public function setIdentidad($identidad)
   {
      $this->identidad = $identidad;
   }

   public function toString() {
      // Retorna una representación en forma de cadena de texto del objeto Denunciante
      return "ID: " . $this->getIdentidad() . ", 
      Nombre: " . $this->getPrimerNombre() . " ". $this->getSegundoNombre() . ", 
      Apellido: " . $this->getPrimerApellido() . " ". $this->getSegundoApellido() . ",
      Edad: "  . $this->getEdad() . ",
      Genero: " . $this->getGenero() . ",
      Dirección: " . $this->objDireccion->toString(); // Llama al método __toString de la clase Direccion
  }
  
  }











?>